code/game/objects/items/shooting_range.dm

```diff
@@ -17,6 +17,13 @@
 	pinnedLoc = null
 	density = 0
 
+/obj/item/target/newtonian_move(angle)
+	if(pinnedLoc)
+		..(angle, TRUE)
+		return FALSE

```

Comment:
Replace this entire proc with 

````
	. = ..()
	if(pinnedLoc)
		. = FALSE
````

Only this proc's return value is used. you don't need to make an argument in the parent's newtonian_move to return false

and if you call ..() without any args, the args are forwarded